src/components/button/Button.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs, text, select, object } from '@storybook/addon-knobs'
+import { action } from '@storybook/addon-actions'
+import styled from 'styled-components'
+import { variant } from 'styled-system'
+import { colors } from '../../theme/colors'
+import { Button } from './Button'
+import { ButtonBaseProps } from './Button.types'
+
+const Wrapper = styled.div<ButtonBaseProps>`
+  padding: 50px;
+  text-align: center;
+
+  ${variant({
+    variants: {
+      primary: {
+        backgroundColor: 'transparent',
+      },
+      secondary: {
+        backgroundColor: 'transparent',
+      },
+      outline: {
+        backgroundColor: 'transparent',
+      },
+      'outline-white': {
+        backgroundColor: colors.indigo,
+      },
+    },
+  })}
+`
+Wrapper.displayName = 'Wrapper'
+
+const variants = {
+  Primary: 'primary',
+  Secondary: 'secondary',
+  Outline: 'outline',
+  'Outline White': 'outline-white',
+} as const
+
+storiesOf('Button', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    () => (
+      <Wrapper variant={select('Variant', variants, variants.Primary)}>
+        <Button
+          onClick={action('click')}
+          variant={select('Variant', variants, variants.Primary)}
+        >
+          {text('Link Text', 'This is a button')}
+        </Button>
+      </Wrapper>
+    ),
+    {
+      info: {
+        text:
+          'By default the Button component will render an html <button />. You can select one of the styles by passing a `variant` prop. The <Wrapper /> in the story source is to provide a contrasting background for the white variant only.',
+        components: { Wrapper: Wrapper },
+      },
+    }
+  )
+  .add(
+    'as Gatsby <Link />',
+    () => (
+      <Button to="/relative-url">{text('Link Text', 'Gatsby Link')}</Button>
+    ),
+    {
+      info: {
+        text:
+          'If given a "to" prop which is a relative URL, it will automatically render a Gatsby <Link /> component',
+      },
+    }
+  )
+  .add(
+    'as external link',
+    () => (
+      <Button to="https://google.com">
+        {text('Link Text', 'External link')}
+      </Button>
+    ),
+    {
+      info: {
+        text:
+          'If given a "to" prop which is an external URL, it will automatically render a html <a /> tag with the target attribute to open in a new tab and rel (noopener and noreferrer) security attributes.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 207. Maximum allowed is 80.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)